website/server/libs/amazonPayments.js

```diff
@@ -98,6 +98,9 @@ api.checkout = async function checkout (options = {}) {
 
   if (gift) {
     if (gift.type === this.constants.GIFT_TYPE_GEMS) {
+      if (gift.gems.amount <= 0) {
+        throw new BadRequest(i18n.t('badAmountOfGemsToPurchase'));
```

Comment:
We've been wanting not to have bare errors use translated strings IIRC--should this error message come from an API errors lib instead?